src/apps/loyalty/server/middlewares/session.ts

```diff
@@ -0,0 +1,11 @@
+import { NextFunction, Request, Response } from "express"
```

Comment:
I called this `SetSession` but its maybe misleading. Basically, we just want to set the session id in sharify, since we're using that to be able to refer to it server and client-side.